lib/material_date_range_picker_dialog.dart

```diff
@@ -33,7 +33,7 @@ class MaterialDateRangePickerDialog {
       enableDrag: false,
       constraints: ResponsiveUtils.isLandscapeMobile(context)
         ? const BoxConstraints(maxWidth: 450)
-        : null,
+        : const BoxConstraints(maxWidth: 9999),
```

Comment:
Why set like that?

Reply:
What if the screen `size is > 9999`? (This latter example runs on TV)

Reply:
How can you find the width limit of your screen? What if the width is greater than 9999?. IMO, there will be a better solution than this.